implementations/contracts/ERC725XCore.sol

```diff
@@ -110,108 +120,94 @@ abstract contract ERC725XCore is OwnableUnset, ERC165, IERC725X {
         uint256 value,
         bytes memory data
     ) internal virtual returns (bytes memory result) {
-        emit Executed(OPERATION_CALL, to, value, bytes4(data));
+        emit Executed(uint256(OPERATION_TYPE.CALL), to, value, bytes4(data));
 
         // solhint-disable avoid-low-level-calls
         (bool success, bytes memory returnData) = to.call{value: value}(data);
         result = Address.verifyCallResult(success, returnData, "ERC725X: Unknown Error");
```

Comment:
@YamenMerhi I am happy to revert back the last commit if we decide to keep the old structure of the internal functions.

The only extra change I would suggest is to change the revert error message passed to the OZ `Address.sol` library to use the same one they have.

So replace for instance `ERC725X: Unknown Error` > `ERC725X: low-level static call failed`

<img width="867" alt="Screenshot 2022-10-21 at 00 23 04" src="https://user-images.githubusercontent.com/31145285/197077415-1100b83d-5734-4fcc-8f53-1e60c0686fa5.png">